test/keypair_test.exs

```diff
@@ -29,4 +29,15 @@ defmodule KeyPairTest do
 
     assert KeyPair.verify_signature(pub, signature, data) == true
   end
+
+  test "can generate an address from keypair" do
```

Comment:
We should start naming the tests very specifically, i.e address generated from key pair matches xxx, may be helpful to start using doc & tags for the tests.

Reply:
Agreed. How do you propose I update this test?